packages/shared/src/lib/timezones.ts

```diff
@@ -0,0 +1,521 @@
+import { utcToZonedTime } from 'date-fns-tz';
+
+const TIME_ZONES = [
```

Comment:
Maybe we can extract this list from `date-fns-tz` by any chance?

Reply:
Nope sorry
https://github.com/marnusw/date-fns-tz/issues/122